alembic/versions/6ff7e38b0060_add_parfumo_url_column_to_fragrances_.py

```
@@ -0,0 +1,31 @@
+"""Add parfumo_url column to fragrances table
+
+Revision ID: 6ff7e38b0060
+Revises: 001
+Create Date: 2025-12-30 15:03:14.929391
+
+"""
+from typing import Sequence, Union
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)